packages/lab/src/index.ts

```diff
@@ -257,7 +257,7 @@ const plugin: JupyterFrontEndPlugin<IRisePreviewTracker> = {
               !document.fullscreenElement &&
               !iframe.contentDocument?.fullscreenElement
             ) {
-              const goFullScreen = () => {
+              const goFullScreen = async () => {
```

Comment:
Left over from testing?

```suggestion
              const goFullScreen = () => {
```